modules/gis/docs/api-reference/geojson-to-binary.md

```diff
@@ -49,24 +48,27 @@ corresponds to 3D coordinates, where each vertex is defined by three numbers.
 {
   points: {
     // Array of x, y or x, y, z positions
-    positions: {value: PositionDataType, size: coordLength},
+    positions: {value: PositionDataType, size: 3},
     // Array of original feature indexes by vertex
     globalFeatureIds: {value: Uint16Array || Uint32Array, size: 1},
     // Array of Point feature indexes by vertex
     featureIds: {value: Uint16Array || Uint32Array, size: 1},
     // Object with accessor objects for numeric properties
     // Numeric properties are sized to have one value per vertex
     numericProps: {
-        numericProperty1: {value: Float32Array, size: 1}
+        numericProperty1: {value: Float32Array || Float64Array, size: 1}
```

Comment:
In typescript, that is normally a single bar `|`. Seems like it needs updating across this example. 
```suggestion
        numericProperty1: {value: Float32Array | Float64Array, size: 1}
```